### PR TITLE
Templates for spider

### PR DESCRIPTION
**Pull request by NohTow**
Templates for the spider dataset.
One major issue was the variation in the writing style of question (some are not really "questions"), so I tried to make it sounds like exercise statements, hope it is good enough.

### CODE REVIEW

File: templates/spider/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+dataset: spider
+templates:
+  4f63f872-06e2-4503-a440-c1908d5cb642: !Template
+    id: 4f63f872-06e2-4503-a440-c1908d5cb642
+    jinja: Can you give me the SQL request which respond to "{{question}}" ? ||| {{query}}
```
Comment (srush) [inline templates/spider/templates.yaml:5]: Respond should be "corresponds"

File: templates/spider/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    name: text2SQL_1
+    reference: ''
+  b4168ce2-4854-4c09-a5aa-08cbcb073ab0: !Template
+    id: b4168ce2-4854-4c09-a5aa-08cbcb073ab0
+    jinja: How would you query a SQL database to answer "{{question}}" ? ||| {{query}}
+    name: text2SQL_4
+    reference: ''
+  ba6415e9-9467-4519-8789-57d65524464e: !Template
+    id: ba6415e9-9467-4519-8789-57d65524464e
+    jinja: The SQL query associated to the question "{{question}}" is ||| {{query}}
```
Comment (srush) [inline templates/spider/templates.yaml:15]: "associated with"

File: templates/spider/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    name: text2SQL_3
+    reference: ''
+  e843ba1e-09af-48ba-92ca-1b6032404985: !Template
+    id: e843ba1e-09af-48ba-92ca-1b6032404985
+    jinja: 'To respond to "{{question}}", I can use the SQL query: ||| {{query}}'
```
Comment (srush) [inline templates/spider/templates.yaml:20]: Instead of "respond to" , I would say "lookup "

**Review by srush — CHANGES_REQUESTED** [review]
These are great. But can you also include the reverse entries. "I would describe the SQL statement ... to a new learner as ||| ..."

**Comment by NohTow** [comment]
I added your corrections/recommendations and checked the main task box.
I also added the reverse entries, feel free to tell me if something is odd.

**Pull request merged**